pkg/apis/core/helper/seed_test.go

```diff
@@ -235,12 +235,18 @@ var _ = Describe("Helper", func() {
 	Describe("#ConvertSeedTemplate", func() {
 		It("should convert the external SeedTemplate version to an internal one", func() {
 			Expect(ConvertSeedTemplate(&gardencorev1beta1.SeedTemplate{
+				ObjectMeta: metav1.ObjectMeta{
```

Comment:
Nit: Could you also add annotations here, just to be very sure? :)

Reply:
Of course, done :)